### PR TITLE
port automated mount for stress / longhaul

### PR DESCRIPTION
**Pull request by and-rewsmith**

### CODE REVIEW

**Comment by varunpuranik** [comment]
Does this PR always mount the volume into the containers? Should we just provide a option to do that instead? So that it can only be enabled for the e2e tests where it makes sense?

File: scripts/windows/test/Run-E2ETest.ps1 [hunk 1]
```diff
@@ -1324,6 +1324,15 @@ Function SetEnvironmentVariable
     $env:Path="$env:Path;C:\Program Files\iotedge-moby;C:\Program Files\iotedge"
 }
 
+Function SetupMountedStorage
```
Comment (varunpuranik) [inline scripts/windows/test/Run-E2ETest.ps1]: Here you don't seem to delete the mount folder if it exists.

File: scripts/linux/runE2ETest.sh [hunk 2]
```diff
@@ -19,6 +19,9 @@ function clean_up() {
     rm -rf /var/run/iotedge/
     rm -rf /etc/iotedge/config.yaml
 
+    echo 'Remove mounted storage folder'
+    rm -rf $mounted_storage_path
```
Comment (varunpuranik) [inline scripts/linux/runE2ETest.sh]: Suggest moving this to setup_mounted_storage

File: scripts/windows/test/Run-E2ETest.ps1 (hunk 1 above)
Comment (and-rewsmith) [reply scripts/windows/test/Run-E2ETest.ps1]: The Setup-Env.ps1 should handle that part

Comment (and-rewsmith) [reply scripts/windows/test/Run-E2ETest.ps1]: But we should probably keep all the logic in one place, so I will move

File: scripts/linux/runE2ETest.sh [hunk 3]
```diff
@@ -852,9 +855,17 @@ function run_test()
     exit $ret
 }
 
+function setup_mounted_storage() {
```
Comment (and-rewsmith) [inline scripts/linux/runE2ETest.sh]: need to fix this brace

Comment (and-rewsmith) [reply scripts/linux/runE2ETest.sh]: Actually, same line is the standard here. run_test() is the weird one

**Comment by and-rewsmith** [comment]
The mount will only happen for stress and long haul, as it is specified as a bind in the deployment file through CreateOptions. The option is a good idea so other e2e tests can take advantage of the mounts. Their deployment files will need to change though if they want to do so.

**Comment by and-rewsmith** [comment]
Closing as we should make configurable.

**Pull request closed without merging**